CONTRIBUTING.md

```diff
@@ -90,7 +90,7 @@ Before you submit your pull request consider the following guidelines:
     mvn spring-boot:run
     ```
 
--   You can also run our travis build locally by following [this](#local-travis-build)
+-   You can also generate the same samples than our Continuous Integration (with Travis CI and Azure Pipelines) by following [this](#local-build)
```

Comment:
can be simplified into `you can generate our Continuous Integration  (with Travis CI and Azure Pipelines) by following [this](#local-build)`